planning/behavior_velocity_traffic_light_module/src/scene.hpp

```diff
@@ -145,6 +145,9 @@ class TrafficLightModule : public SceneModuleInterface
 
   // Traffic Light State
   autoware_auto_perception_msgs::msg::LookingTrafficSignal looking_tl_state_;
+
+  std::shared_ptr<motion_utils::VirtualWallMarkerCreator> virtual_wall_marker_creator_ =
+    std::make_shared<motion_utils::VirtualWallMarkerCreator>();
```

Comment:
I think we do not need a `shared_ptr` here.
```suggestion
  motion_utils::VirtualWallMarkerCreator virtual_wall_marker_creator_;
```

Reply:
@maxime-clem 
Thank you for the comment!
I declared without pointer!